include/pops/date.hpp

```diff
@@ -443,6 +444,29 @@ int Date::weeks_from_date(Date start)
     return week - 1;
 }
 
+/*!
+ * Returns the current date as a string
+ * \param Date, current date
+ * \return String, current date as string
+ */
+std::string Date::to_string()
+{
+    std::string date = std::to_string(year_) + "-";
+    if (month_ < 10) {
+        date = date + "0" + std::to_string(month_) + "-";
+    }
+    else {
+        date = date + std::to_string(month_) + "-";
+    }
+    if (day_ < 10) {
+        date = date + "0" + std::to_string(day_);
+    }
```

Comment:
The current C++ solution for these would be a string stream, but that's actually less readable for non-C++ programmers than this, so just note in the code that this is to be replaced by the C++20 format function in the future.